src/main/java/org/usfirst/frc4904/robot/commands/FourBarSet.java

```diff
@@ -0,0 +1,28 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.subsystems.FourBarLinkage.FourBarState;
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ * Control elevator manually
+ */
+public class FourBarSet extends Command {
```

Comment:
Should use a command group to set the state of elevator and the four bar. Ensure the isFinished times out instead of being false (due to the solenoid set thing)